include/persistent_data.hpp

```diff
@@ -188,6 +189,84 @@ class ConfigFile
         }
     }
 
+#ifdef BMCWEB_ENABLE_IBM_MANAGEMENT_CONSOLE
```

Comment:
Anyway we could use `void writeData()`  ?

Reply:
since this is specific to session data used this method name writeCurrentSessionData

Reply:
Is a lot of dupped code :/

Reply:
I wonder if we should get rid of the ifdef here.. upstream bmcweb has been doing that 

We would keep the ifdefs around 

```
#ifdef BMCWEB_ENABLE_IBM_MANAGEMENT_CONSOLE
        signals.add(SIGUSR1);
#endif
```


```
#ifdef BMCWEB_ENABLE_IBM_MANAGEMENT_CONSOLE
                if (signalNo == SIGUSR1)
                {
                    BMCWEB_LOG_CRITICAL


```